ui/components/component-library/component-library-components.scss

```diff
@@ -4,6 +4,12 @@
 * This will help improve specificity and reduce the chance of
 * unintended overrides.
 **/
+
+// Temp CSS Variables
+:root {
+  --opacity-disabled: 0.5;
+}
+
```

Comment:
I like this idea but we may need to have a discussion about where we keep these global variables. I suppose here for the time being is ok? The should be used outside of the component-library folder as well. Also CSS variables vs Sass variables etc

Reply:
Duplicate comment you've resolved in other comment sorry